src/index.js

```diff
@@ -507,7 +507,11 @@ class PhoneInput extends React.Component {
       if (value.slice(0, updatedInput.length) !== updatedInput) return;
     }
 
-    if (value === prefix) return this.setState({ formattedNumber: '' });
+    if (value === prefix) {
+      // we should handle change when we delete the last digit
+      if (onChange) this.props.onChange('', this.getCountryData(), e, '');
```

Comment:
I guess, it's better to use desctructured onChange here if you check it in condtion
```suggestion
      if (onChange) onChange('', this.getCountryData(), e, '');
```

It would be even better with optional chaining, but I'm not sure that this feature is enabled here.
```suggestion
      onChange?.('', this.getCountryData(), e, '');
```

Reply:
Thanks for reviewing. I used destructured onChange. Unfortunately chaining is not support here